### PR TITLE
Feat: Add extras to index route

### PR DESCRIPTION
**Pull request by ghost**
Adds extra information about the validator in the index route, also add status route

### CODE REVIEW

File: src/server/mod.rs [hunk 1]
```diff
@@ -51,13 +53,13 @@ where
...
                 .wrap(Logger::default())
-                .route("/", web::get().to(index))
+                .route("/", web::get().to(index::<Context, KeyManager>))
                 .route("/tx/{tx_id}", web::get().to(get_tx::<Context>))
                 .service(
                     web::scope("/cosigner")
                         .route("/sign", web::post().to(sign_route::<Context, KeyManager>)),
                 )
-                .service(web::scope("/idle").route("/", web::get().to(index)));
+                .service(web::scope("/idle").route("/", web::get().to(index::<Context, KeyManager>)));
```
Comment (mikonieminen) [inline src/server/mod.rs:64]: What is purpose of this? Where would it be used?

File: src/server/routes/status.rs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+#[derive(Serialize)]
+struct StatusBody {
+    total_txs: i64,
+    epoch: u128,
+    next_epoch: u128,
+    previous_epoch: u128
+}
+
+pub async fn index<Context, KeyManager>(
```
Comment (mikonieminen) [inline src/server/routes/status.rs]: Maybe this should be renamed as `status`.

File: src/server/mod.rs [hunk 3]
```diff
@@ -51,13 +53,14 @@ where
             let app = App::new()
                 .app_data(Data::new(runtime_context.clone()))
                 .wrap(Logger::default())
-                .route("/", web::get().to(index))
+                .route("/", web::get().to(index::<Context, KeyManager>))
+                .route("/status", web::get().to(index::<Context, KeyManager>))
```
Comment (mikonieminen) [inline src/server/mod.rs]: Shouldn't this call `index` from status? Or am I missing something?

File: src/server/mod.rs (hunk 1 above)
Comment (ghost) [reply src/server/mod.rs:64]: Checkout: 
https://node1.bundlr.network/
https://node1.bundlr.network/public

It comes with some useful information and stats for monitoring and integrations

File: src/server/routes/status.rs (hunk 2 above)
Comment (ghost) [reply src/server/routes/status.rs]: Yes, mb

File: src/server/mod.rs (hunk 3 above)
Comment (ghost) [reply src/server/mod.rs]: mb again

File: src/server/mod.rs (hunk 1 above)
Comment (ghost) [reply src/server/mod.rs:64]: The goal here is to add similar information to the "/" (index) and "/status" routes

Comment (mikonieminen) [reply src/server/mod.rs:64]: Yeah, but why two different entrypoints with same data? Or is the purpose just to have these entrypoints in place and then later provide different data?

Comment (ghost) [reply src/server/mod.rs:64]: Yes, later provide different data

Comment (mikonieminen) [reply src/server/mod.rs:64]: Oh, actually this was not added. It was just changed to match signature of `index`. My bad.

**Review by mikonieminen — APPROVED** [review]

**Pull request merged**